eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -9,8 +9,6 @@ jobs:
     - subset: coreclr
       include:
       - src/libraries/System.Private.CoreLib/*
-      - src/libraries/Native/Unix/System.Globalization.Native/*
-      - src/libraries/Native/Unix/Common/*
       - src/native/*
```

Comment:
should we exclude from the coreclr path partition the shims that are not affecting/linked into the runtime? i.e: 
exclude: `src/native/libs/*` but include :`src/native/System.Globalization.Native/*` and whatever common paths are shared for `Globalization.Native?`

Reply:
The depndencies in native libs and other libs code is not that striaightforward. CoreCLR build parts of libs as well as corehost, the latter then relies on parts of libs. I wonder if it is better to build all related subsets when something changes in src/native/*? cc @VSadov